packages/prisma/zod-utils.ts

```diff
@@ -220,6 +220,7 @@ export const bookingCreateBodySchema = z.object({
   eventTypeSlug: z.string().optional(),
   rescheduleUid: z.string().optional(),
   recurringEventId: z.string().optional(),
+  rescheduledBy: z.string().email({ message: "Invalid email" }).optional(),
```

Comment:
```suggestion
  rescheduledBy: z.union([
    z.string().email({ message: "Invalid email" }),
    z.literal('Anonymous')
  ]).optional(),
```

Should I change it to something like this since we are defaulting to Anonymous or should I use Anonymous@example.com as default?

Reply:
Yeah, not a fan of this 🤔 
Let's resort to `null` instead. So the zod validation would need to be updated as well. 🙏